base-system/usrroot/usr/lib/systemd/system/ipman.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Set IP address on the first interface that has ping to the directives server
+
+[Service]
+ExecStart=/usr/sbin/ipman
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
What about `After=network.target` so that we're sure that we're running after network target. 
Also we could also make a dependency with Connman service directly.

Reply:
You're right, Im kinda new on the services, so this flew over my head